client/extensions/woocommerce/state/sites/customers/selectors.js

```diff
@@ -1,86 +1,58 @@
-/**
- * External dependencies
- */
-
-import { filter, get } from 'lodash';
-
 /**
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
 
+function inFlight( state, siteId, searchTerm ) {
```

Comment:
Now that this function is outside and acts as an actual selector, should it have a more descriptive name?

Reply:
Good point. Fixed.